### PR TITLE
Resolve TS errors: ManageProjects

### PR DESCRIPTION
**Pull request by prachigarg19**
This PR resolves ts errors for all files in` ManageProjects`

### CODE REVIEW

File: src/features/user/ManageProjects/components/ProjectMedia.tsx
Comment (mohitb35) [inline src/features/user/ManageProjects/components/ProjectMedia.tsx:1]: While fetching images for the project, you're using the `Project` type. I think this should be handled similar to `ProjectCertificates` - where you've defined a `CertificateScopeProjects` interface.

```
const fetchImages = async () => {
    try {
      // Fetch images of the project
      if (projectGUID && token) {
        const result = await getAuthenticatedRequest<Project>(
```

File: src/features/common/types/project.d.ts
Comment (mohitb35) [inline src/features/common/types/project.d.ts:1]: You've made a change here to `ProjectSitesProps`, but ProjectSites has not been updated (handle other issues over there as well)

File: src/features/user/ManageProjects/components/ProjectSpending.tsx
Comment (mohitb35) [inline src/features/user/ManageProjects/components/ProjectSpending.tsx:1]: While fetching expenses for the project, you're using the `Project` type. I think this should be handled similar to `ProjectCertificates` - where you've defined a `CertificateScopeProjects` interface.

File: src/features/user/ManageProjects/index.tsx [hunk 1]
```diff
@@ -358,7 +334,11 @@ export default function ManageProjects({
 
   return (
     <DashboardView
-      title={projectGUID ? project?.name : t('manageProjects:addNewProject')}
+      title={
+        project && projectGUID
+          ? project?.name
```
Comment (mohitb35) [inline src/features/user/ManageProjects/index.tsx:339]: The ? check in `project?.name` is redundant

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@prachigarg19 Some minor changes suggested, but there are 2 components which seem to have been left alone: `MapComponent` and `ProjectSites`. Could you address the TS warnings there too?

**Comment by prachigarg19** [comment]
> @prachigarg19 Some minor changes suggested, but there are 2 components which seem to have been left alone: `MapComponent` and `ProjectSites`. Could you address the TS warnings there too, if they are not being addressed in #1856 ?

@sunil-sabat will be working on `MapComponent`
Error: `Unexpected any. Specify a different type.` was not resolved since currently I'm facing `500` error when I try to save a site.

**Review by mohitb35 — APPROVED** [review]
LGTM

**Pull request merged**